assembl/static2/js/app/components/administration/brightMirror/load.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import sortBy from 'lodash/sortBy';
+import type { ApolloClient } from 'react-apollo';
+
+import ThematicsQuery from '../../../graphql/ThematicsQuery.graphql';
+import { convertEntries } from '../../form/utils';
+import { convertEntriesToRawContentState } from '../../../utils/draftjs';
+import type { BrightMirrorAdminValues } from './types.flow';
+
+export const load = async (client: ApolloClient, fetchPolicy: FetchPolicy) => {
+  const { data } = await client.query({
+    query: ThematicsQuery,
+    variables: { identifier: 'brightMirror' },
```

Comment:
Please add the brightMirror phase id in the PHASES constant and then use it (PHASES.brightMirror)